docs/guides/debugging/crabnebula-devtools.md

```diff
@@ -0,0 +1,37 @@
+# Debugging in CrabNebula DevTools
+
+CrabNebula provides a free DevTools application for Tauri as part of its partnership with the Tauri project.
```

Comment:
```suggestion
[CrabNebula](https://crabnebula.dev) provides a free [DevTools](https://crabnebula.dev/devtools/) application for Tauri as part of its partnership with the Tauri project.
```

Reply:
hmm, thinking back to the partners overview/intro page we want to have, do you think it makes sense to swap out the links once that's live?

Reply:
I like that idea